CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.2.1
```

Comment:
Have to make this like `Upcoming Changes` cuz I checked and there are a lot more fields for different platforms to be changed.

So I'll publish them all together.

Reply:
Resolved in [new commit](https://github.com/obemu/package_rename/commit/23fccf09eeb92780e0bd0e10b45c231503e02128)